### ISSUE TITLE
Tabs component not working

### ISSUE THREAD

**Issue by minglho** [isle-project/isle-editor#1131]
The Tabs component showed up in the autocompete, so I clicked on it, and some template code showed up. However, the code didn't render, as seen below. The complete error message on the right is:

> ReactBootstrap: The `Tab` component is not meant to be rendered! It's an abstract component that is only valid as a direct Child of the `Tabs` Component. For custom tabs components use TabPane and TabsContainer directly

![image](https://user-images.githubusercontent.com/13249661/87124506-c741e700-c23d-11ea-8315-4c3fa618cc43.png)

What I wanted to do is something as shown below. If the Tabs component isn't it, is there an alternative?

![image](https://user-images.githubusercontent.com/13249661/87125052-b180f180-c23e-11ea-844e-1cf5b0d7f353.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - ISLE v.0.57.1

**Comment by minglho** [comment]
I suppose I can use the Pages component, but I really would like to provide a title or each page rather than just 1, 2, 3....

**Comment by Planeshifter** [comment]
Thanks for reporting this issue, which was caused due to the recent instrumentation that allows one to jump between to elements in the source view and preview via double-clicking on them. A fix is included in 26099b52da109f0446f4696397093ceccb3a603c, which will be included in the next version of the editor. Plan is to have it out by end of next week at latest. If that is not sufficient, one option would be to clone the git repository and build the editor yourself.

The <Tabs /> component is the right solution for what you are trying to achieve. For example,

```
<Panel>
    <Tabs defaultActiveKey={1} >
        <Tab eventKey={1} title="Tab 1">Tab 1 content</Tab>
        <Tab eventKey={2} title="Tab 2">Tab 2 content</Tab>
        <Tab eventKey={3} title="Tab 3"  >Tab 3 content</Tab>
    </Tabs>
</Panel>
```

should render as follows:

![image](https://user-images.githubusercontent.com/1913638/87186232-2264ef80-c2b9-11ea-8a60-a4edc6a5b059.png)

Even with v0.57.1, the Tabs should render correctly once the lesson is uploaded. The recent bug affects the preview in the editor only, although that's not much of a relief.

**Comment by Planeshifter** [comment]
Closing as this issue is now resolved.
